nilearn/plotting/displays.py

```diff
@@ -1242,37 +1242,45 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
                           for c in sorted(self._cut_displayed)]
         return cut_coords
 
+    def _find_inital_axes_coord(self,index):
+        "find coordinates for inital axes placement for xyz cuts"
```

Comment:
Docstrings in Python should be Triple double quotes, i.e.: 
```
""" Find coordinates...
"""
```

Docstrings should explain what parameters are expected. Here: index, datatype, what is it.
They should also explain what is being returned. Here List(coords). What are cords?

Check out Docstrings in existing functions to see how to format them. That is especially important as we use automated tools to parse & read the docstrings and insert them into our documentation. The tools expect the docstrings to follow very strict formatting rules.